### DISCUSSION TITLE
Server takes a long time to start in case of too many models

### DISCUSSION THREAD

**Discussion by aaqilniz** [loopbackio/loopback-next#7348] [Q&A]
I have a database with around 2000 tables. I am trying to serve with REST APIs and I am trying to achieve this by using LoopBack 4's Autodiscovery. I understand having this big number of tables will have an effect on the server but is there any way to improve the performance of the server & decrease the time it takes to start?

**Comment by raymondfeng** [comment]
Do you run discovery as part of server startup?

**Reply by aaqilniz** [reply]
No. I actually have a script that discovers & creates models - which is done before starting the server. 
All of the models are created before starting the server. The server still takes too long to start (around 1.5 to 2.5 hours!)
